tests/test_trusted-execution.yml

```diff
@@ -0,0 +1,132 @@
+---
+- name: Basic sanity test for fapolicyd
+  hosts: localhost
+  vars:
```

Comment:
Prefix vars with `__bootloader` to avoid conflict with variable names set in other places. Not very likely for tests but still a best practice. E.g. change `bindaries_directory` to `__bootloader_binaries_dir`.